src/recipe/test/unit/recipe.service.spec.ts

```diff
@@ -129,6 +120,8 @@ describe('Recipe Service', () => {
     expect(result.ingredients[0].name).toBe('flour');
     expect(result.ingredients[0].amount).toBe('spoon');
     expect(result.authorId).toBe(1);
+
+    queryBusExecuteFindOne.mockClear();
```

Comment:
if you need to clear / restore mock the best (IMO) way would be to add mock and clearing in beforeEach/afterEach

example:
`  afterEach(() => {
    jest.restoreAllMocks();
    jest.clearAllMocks();
  });`
  
`
    beforeEach(() => {
      jest.spyOn(fieldService, 'findFieldIdsByCropIds').mockImplementation(async () => null);
    });
`